oddbird/Metecho
# Allow converting Task Scratch Org into Dev Org.

## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

![](https://source.unsplash.com/featured/?cute,animal%210604)

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/1wrCiCiM/233-i-want-to-convert-a-task-scratch-org-into-a-dev-org-for-that-task

## Description
_The commit messages say what you did; this should explain why and/or how._

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- Create a **Task** Scratch Org
- Make changes in the org
- Click "contribute" btn on org
- Select either "Convert Scratch Org into Dev Org on a new Task" or "Convert Scratch Org into Dev Org for this Task" (disabled if there is already a Dev Org on the task)
- Click "Contribute" btn
- If you selected "new task", fill out form, then submit and see the new task with your org as the Dev Org (and you assigned as the Dev), then the "retrieve changes" modal opens
- If you selected "this task", see your org now as the Dev Org (and you assigned as the Dev, if you weren't before), then the "retrieve changes" modal opens

## Show me
_Provide screenshots/animated gifs/videos if necessary._

![Screen Shot 2021-06-04 at 3 51 30 PM](https://user-images.githubusercontent.com/552316/120855808-f398dd00-c54c-11eb-83b4-07011bca2c83.png)

![Screen Shot 2021-06-04 at 3 51 36 PM](https://user-images.githubusercontent.com/552316/120855811-f4317380-c54c-11eb-82f0-bdf2e8ee0195.png)

![Screen Shot 2021-06-04 at 3 52 40 PM](https://user-images.githubusercontent.com/552316/120855812-f4317380-c54c-11eb-890e-f52c0072e7b3.png)
